server/database/connection.js

```diff
@@ -1,35 +1,44 @@
 const express           = require('express');
 const _dotenv           = require('dotenv').config();
```

Comment:
Can delete these 2 unused imports

Reply:
Why is line 2 not being used for the getConnectionURI function validation on lines 9-13?